Adding documentation for the value (hiddenMembership) for team visibility type (#19724)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/teamvisibilitytype.md

|:---------------|:--------|:----------|
|private|0|Anyone can see the team but only the owner can add a user to the team.|
|public|1|Anyone can join the team.|

> |hiddenMembership|2|Only the administrators (global, company, user and helpdesk) can view the members of the team.<br>Owner permission is needed to join the team.|

Reviewer: ```suggestion
|hiddenMembership|2|Only the administrators (global, company, user, and helpdesk) can view the members of a team.<br>Owner permissions are required to join a team.|
```
